samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/print/PDFAssessmentBean.java

```diff
@@ -390,7 +390,9 @@ public String prepDocumentPDF() {
 				if (!(TypeIfc.FILL_IN_BLANK.equals(item.getItemData().getTypeId()) || TypeIfc.FILL_IN_NUMERIC.equals(item.getItemData().getTypeId())  
 					    || TypeIfc.CALCULATED_QUESTION.equals(item.getItemData().getTypeId()))) {
 					contentBuffer.append("<br />");
-					contentBuffer.append(convertFormattedText(item.getItemData().getText()));
+					if (item.getItemData().getText() != null) {
```

Comment:
Instructions on hot spot questions can be null on Oracle servers.

Reply:
This could probably be improved with StringUtils.isNotBlank as there wouldn't be a need to call convertFormattedText with an empty String.